utils/dbdump/init.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is executed at run time during container initialization / entrypoint
+echo "[Entrypoint] Database dump started";
+echo "[Entrypoint] Databse dump mode: $DATABASE_DUMP_MODE";
+if [ "$DATABASE_DUMP_MODE" == "big" ]; then
```

Comment:
nit: prefer `[[]]` over `[]` ([style guide](https://google.github.io/styleguide/shellguide.html#s6.3-tests))

Reply:
fixed this, thanks!